business_requirement_deliverable/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    pricelist_id = fields.Many2one(
```

Comment:
@victormartinelicocorp I remember you told me the Sales Pricelist was a computed field. Reason is: when the partner is a contact of a company, his Sales Pricelist is the one from his company. As discussed, we should replicate this behavior.